packages/core/src/App/Containers/RealAccountSignup/signup-error-content.jsx

```diff
@@ -102,6 +102,8 @@ const ErrorCTA = ({ code, onConfirm }) => {
             return <TryAgain text={localize('Try a different currency')} onConfirm={onConfirm} />;
         case 'DuplicateAccount':
             return null;
+        case 'DuplicateCurrency':
+            return <TryAgain text={localize('Try a different currency')} onConfirm={onConfirm} />;
```

Comment:
Can't we just combine it with `case 'CurrencyTypeNotAllowed'` as the returned component is same or I'm missing something!